### PR TITLE
UI-515: Text Component

### PR DESCRIPTION
**Pull request by kokasvmx**

### CODE REVIEW

File: lib/components/Text/Text.js [hunk 1]
```diff
@@ -2,18 +2,76 @@ import React from 'react';
...
+  /**
+   * Classname for section title divider
+   */
+  section: PropTypes.bool,
+  /**
+   * Element to be added 
+   */
+  tag: PropTypes.string,
+  /**
+   * Classname for tuncate
```
Comment (mallik-svmx) [inline lib/components/Text/Text.js]: minor typo tuncate

File: lib/components/Text/Text.js [hunk 2]
```diff
@@ -2,18 +2,76 @@ import React from 'react';
...
+   */
+  truncate: PropTypes.bool,
+  /**
+   * Type of the text
+   */
+  type: PropTypes.oneOf(TEXT_TYPES),
+};
+
+const defaultProps = {
+  category: 'body',
```
Comment (joshsylvester) [inline lib/components/Text/Text.js]: need defaults for `align` `section` and `truncate`

File: lib/components/Text/Text.js [hunk 3]
```diff
@@ -2,18 +2,76 @@ import React from 'react';
...
+  align,
+  truncate,
+  section,
+  children,
+  className,
+  ...props
+}) => {
   const textClassNames = classnames(
     {
+      'slds-section-title--divider': section,
```
Comment (joshsylvester) [inline lib/components/Text/Text.js]: Please add a component BEM classname - `Text`

File: lib/components/Text/Text.test.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import Text from './../Text';
+import { Text } from './../Text';
```
Comment (joshsylvester) [inline lib/components/Text/Text.test.js]: `import Text from './Text';`

**Pull request merged**